packages/ui-predicate-core/src/index.js

```diff
@@ -1,4 +1,5 @@
 const errors = require('./errors');
+const UI = require('./ui');
```

Comment:
Le fichier ui n'est pas présent dans la PR ? si ?

Il faudra ajouter cela en haut du "ui" pour que la doc puisse bien build : 

/**
 * Rules
 * @module ui
 * @namespace core
 * @since 1.0.0
 * @note default (and overridable) base ui components of ui-predicate (e.g. target select, operator select, add button, remove button, ...)
 */